filters/src/main/java/io/camunda/zeebe/process/test/filters/logger/RecordStreamLogger.java

```diff
@@ -112,6 +112,7 @@ public RecordStreamLogger(final RecordStreamSource recordStreamSource) {
     valueTypeLoggers.put(ValueType.RESOURCE_DELETION, this::logResourceDeletionRecordValue);
 
     valueTypeLoggers.put(ValueType.COMMAND_DISTRIBUTION, this::logCommandDistributionRecordValue);
+    valueTypeLoggers.put(ValueType.PROCESS_INSTANCE_BATCH, record -> "");
```

Comment:
:thought_balloon: I assume we don't log the `ProcessInstanceBatch` records since they are not relevant for testing (and there really isn't any relevant information to log) correct?

Reply:
Exactly, there isn't really anything in it that's relevant for the user